models/User.php

```diff
@@ -57,6 +58,11 @@ public function rules()
             ['status', 'default', 'value' => self::STATUS_ACTIVE],
             ['status', 'in', 'range' => [self::STATUS_ACTIVE, self::STATUS_DELETED]],
             ['status', 'filter', 'filter' => 'intval'],
+            
+            ['access_token', 'string', 'length' => 64],
```

Comment:
That would allow setting user token by editing your profile. That's not desired effect.

Reply:
Since token is never set directly, I'd either remove rules for it or redefine default scenario.

Reply:
Fixed.